1.10.5/rhel7/Dockerfile

```diff
@@ -64,7 +64,8 @@ RUN pip3.6 install --no-cache-dir cryptography  \
      &&  pip3.6 install --no-cache-dir "https://github.com/astronomer/astronomer-airflow-scripts/releases/download/v0.0.4/astronomer_airflow_scripts-0.0.4-py3-none-any.whl" \
      &&  cd /usr/local/lib/python3.6/site-packages/airflow/www_rbac \
      &&  /node-v12.13.1-linux-x64/bin/npm install \
-     &&  /node-v12.13.1-linux-x64/bin/npm run build
+     &&  /node-v12.13.1-linux-x64/bin/npm run build \
+     && update-alternatives --install /usr/bin/python3 python3 $(which python3.6) 0
```

Comment:
@ianstanton minor style thing but it looks like all the other commands here have double space between the ampersands and the command. I think we use a single space everywhere else like you have. Maybe we should standardize them all here.

Reply:
Do you prefer all single or all double?

Reply:
I think we do single in the other ones. Mostly all about consistency haha.